### ISSUE TITLE
Prediction 100% ?

### ISSUE THREAD

**Issue by YuChenBing** [AlexeyAB/darknet#1433]
Hi, @AlexeyAB 
Yesterday I used your fork [https://github.com/AlexeyAB/darknet](url) to train my own dataset (captured by **drone**)about **2500 interations** on the cloud server, using command `nohup ./darknet detector train data/drone.data cfg/yoloDrone.cfg darknet53.conv.74 -gpus 0 -dont_show >> log.txt 2>&1 & `

But when I use the command `./darknet detector test data/drone.data cfg/yoloDrone.cfg yoloDrone_2500.weights -dont_show -ext_output < train.txt > result.txt` on my PC(MacOS10.12 )

I got the result like this:

<img width="634" alt="screen shot 2018-08-17 at 3 35 41 pm" src="https://user-images.githubusercontent.com/32861958/44254023-18a0f180-a234-11e8-9304-ac67ce5ccadd.png">

![predictions](https://user-images.githubusercontent.com/32861958/44253823-6832ed80-a233-11e8-8817-6bac5b028403.jpg)

Here is a list of **details** :
1. Cloud server info
2. Makefile
3. folder data/obj(including images and label files)
4. label file
5. train.txt
6. .data file 
7. .name file
8. .cfg file

**1. Cloud server info**
Intel Xeon E5-2682v4, GPU:1 * NVIDIA P100, 8 vCPU
ubuntu 16.04

**2. Makefile:**
GPU=1
CUDNN=1
CUDNN_HALF=0
OPENCV=1
AVX=0
OPENMP=0
LIBSO=0

**3. Folder data/obj(including images and label files)**
0000002_00005_d_0000014.jpg  9999945_00000_d_0000135.jpg   9999972_00000_d_0000025.jpg
0000002_00005_d_0000014.txt  9999945_00000_d_0000135.txt   9999972_00000_d_0000025.txt
......

**4. Label file**
1 0.651 0.5623333333333334 0.002 0.006
6 0.66525 0.5263333333333333 0.0105 0.015333333333333332
5 0.662 0.5166666666666666 0.006 0.008
.......

**5. train.txt**
data/obj/0000002_00005_d_0000014.jpg
data/obj/0000002_00448_d_0000015.jpg
data/obj/0000003_00231_d_0000016.jpg
.......

**6 .data file**
classes= 12
train  = /home/AlexeyAB_darknet/data/train.txt
valid = /home/AlexeyAB_darknet/data/val.txt
names = /home/AlexeyAB_darknet/data/drone.names
backup = backup

**7 .name file**
ignored regions
pedestrian
people
bicycle
car
van
truck
tricycle
awning-tricycle
bus
motor
others

**8. cfg file** 
[net]
#Training
**batch=64
subdivisions=8**
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 30000
policy=steps
**steps=10000,15000**
scales=.1,.1

........
[convolutional]
size=1
stride=1
pad=1
filters=51
activation=linear

[yolo]
mask = 0,1,2
**anchors** = 4.0397,8.1347, 5.9649,16.2151, 16.0946,15.7545, 10.3262,28.0045, 29.6906,27.7179, 20.9687,47.7583, 52.9142,47.8410, 37.3306,79.2094, 88.6776,157.5262
**classes**=12
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
(I recalculated the anchors using `./darknet detector calc_anchors data/drone.data -num_of_clusters 9 -width 416 -height 416 -show`)

Can you help me out? I appreciate of that  :)
Thanks!

**Comment by AlexeyAB** [comment]
@CoderNice Hi,

* Try to check your dataset by using: https://github.com/AlexeyAB/Yolo_mark
* Can you show the end of training `log.txt`? What was avg loss?
* What mAP can you get on the cloud server `darknet detector map` ...? https://github.com/AlexeyAB/darknet#when-should-i-stop-training
